src/dmd/parse.d

```diff
@@ -7271,6 +7271,15 @@ final class Parser(AST) : Lexer
             case TOK.rightParentheses:
                 break;
 
+            case TOK.at:
+                Token* pastAttr;
+                if (skipAttributes(t, &pastAttr))
+                {
+                    t = pastAttr;
+                    goto default;
+                }
+                break;
```

Comment:
This is not covered it seems ?

Reply:
I think it needs a fail compile test to cover that.  I'll add one